app/main.py

```diff
@@ -1,3 +1,75 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other_distance: Distance | float) -> Distance:
+        if isinstance(other_distance, Distance):
+            return Distance(self.km + other_distance.km)
+        return Distance(self.km + other_distance)
+
+    def __iadd__(self, other_distance: Distance | float) -> Distance:
+        if isinstance(other_distance, Distance):
+            self.km += other_distance.km
+            return self
+        self.km += other_distance
+        return self
+
+    def __mul__(self, other_distance: float) -> Distance:
+        return Distance(self.km * other_distance)
+
+    def __truediv__(self, other_distance: float) -> Distance:
+        return Distance(round(self.km / other_distance, 2))
+
+    def __lt__(self, other_distance: Distance | float) -> bool:
+        if isinstance(other_distance, Distance):
+            if self.km < other_distance.km:
+                return True
+            return False
+        if self.km < other_distance:
+            return True
+        return False
```

Comment:
```suggestion
        return self.km < other_distance
```
such constructions could be simplified
please fix everywhere